src/App.tsx

```diff
@@ -1,13 +1,10 @@
-import { useState } from 'react'
-
 import './App.css'
+import UserProfile from '../src/containers/UserProfile.tsx'
```

Comment:
don't need to '../src/containers/..`, you can use relative position from the current file (`App.tsx`) location, like `./containers/...`